src/main/main.c

```diff
@@ -163,6 +171,131 @@ void listen_to_buttons (void *pvParameters) {
     }
 }
 
+esp_bd_addr_t trusted_device;
+bool paired = false;
+
+static char *bda2str(esp_bd_addr_t bda, char *str, size_t size) {
+    if (bda == NULL || str == NULL || size < 18) {
+        return NULL;
```

Comment:
Vil bare lige advare at NULL ikke eksistere i C/C++, du vil returnere og eller tjekke for tallet 0. vær sikker på at det ikke er et scenarie som give problemer

Reply:
Same as before.